### ISSUE TITLE
Aria label is missing for the months grid section of Calendar component

### ISSUE THREAD

**Issue by fxvarga** [microsoft/fluentui#15074]
https://github.com/microsoft/fluentui/blob/a206613fdde9717a5c1f77a57368d83916a608d7/packages/office-ui-fabric-react/src/components/Calendar/CalendarMonth.tsx#L206

We recently had an accessibility review on an app that is utilizing the DatePicker component.  We had a finding for missing a caption for the months grid of the months selector section of the DatePicker. <div class="ms-DatePicker-optionGrid optionGrid_b00ea163" role="grid" aria-readonly="true">. Since this component works as a table, a caption is required to describe the table so that a screen reader will be able to help the user know what this table is about. Please let us know if there is something that we are missing or if this component is indeed missing the aria-label

**Comment by paulgildea** [comment]
@fxvarga Thanks for the feedback.

Could you let us know what version of the `DatePicker` component you're using? Just to make sure there wasn't a regression.

@smhigley - FYI on the recommendation of the described approach
@lorejoh12 - Just a heads up here. I know you've been doing a11y fixes in this area here.

**Comment by fxvarga** [comment]
@paulgildea We are using the DatePicker component that comes with office-ui-fabric-react@7.123.8

**Comment by ecraig12345** [comment]
FYI for whoever ends up working on this issue: in version 8, Calendar is being replaced with the one from `@fluentui/react-date-time` (was `@uifabric/date-time` in version 7). In some cases the new version is better for accessibility, but it looks like this issue exists (and may need to be fixed) in both versions.

**Comment by ecraig12345** [comment]
In case anyone finds this, it should be fixed in version 8 (or `@uifabric/date-time` in version 7). If there are other related problems feel free to open a new issue.
